review/tests.py

```diff
@@ -51,3 +65,29 @@ def test_delete_related_dogowner(self, test_review):
         dogowner.delete()
         assert dogowner not in DogOwner.objects.all()
         assert test_review in Review.objects.all()
+
+    def test_get_review_by_daycare_id(self, test_daycare, test_dogowner):
+        reviews = Review.get_review_by_daycare_id(test_daycare.id)
+        assert len(reviews) == 0
+
+        review_list = []
+
+        review_list.append(Review.create(review='review 1', rating=1, daycare_id=test_daycare.id,
+                                         dogowner_id=test_dogowner.id))
+
+        reviews = Review.get_review_by_daycare_id(test_daycare.id)
+        assert len(reviews) == len(review_list)
+        assert reviews.first() == review_list[0]
+
+        review_list.append(
+            Review.create(review='review 2', rating=2, daycare_id=test_daycare.id, dogowner_id=test_dogowner.id))
+        review_list.append(
```

Comment:
can use `review_list.extend` and add both entries at the same time.

Reply:
actually, not only I suggest you to use `extend`, but IMO you should also parametrize this test instead of testing the same method multiple times on the same test.

Reply:
> actually, not only I suggest you to use `extend`, but IMO you should also parametrize this test instead of testing the same method multiple times on the same test.

what do you mean?

Reply:
I changed to `extend` regarding the parametrize can you share an example of what you thinking about?

Reply:
A test should typically have 3 stages: `arrange`, `act`, `assert`. but here you act -> assert -> arrange -> act ->  assert ->  arrange -> act -> assert.
That's because you're testing 3 different test-cases here (no reviewers, 1 review, 2 reviews).
You can parametrize the test, and so have each of the use cases tested separately; this will keep the test atomic, clean, and readable with only one appearance of each  test case